src/runtime_src/core/include/xgq_impl.h

```diff
@@ -533,6 +533,7 @@ static inline int xgq_attach(struct xgq *xgq, uint64_t flags, uint64_t io_hdl, u
 		// Write 1 to GCQ interrupt enable register to always enable interrupt
 		xgq_reg_write32(xgq->xq_io_hdl, sqprod + XGQ_INTR_ENABLE_OFFSET, 1);
 	}
+
```

Comment:
please don't change the xgq_impl.h unless you have to. The vmr build.sh will check if the xgq_impl.h in vmr repo is matching with XRT version.

Reply:
noted thank you